Jenkinsfile

```diff
@@ -80,7 +80,9 @@ pipeline {
             }

```

Comment:
For what it is worth, you can probably try the Deploy To Test step before merging into master. I think if you just change the `branch 'master'` above on line 79 to `branch '{BranchName}'` then the Deploy to Test step will run when you push to that branch. 

Obviously, you'd need change it back to master before you merge this.

Reply:
Thank worked, force-pushing the commit off the branch